.gitignore

```diff
@@ -39,7 +39,7 @@ MANIFEST
 pip-log.txt
 pip-delete-this-directory.txt
 
```

Comment:
```suggestion
.idea/
```

Reply:
Must ignore editor files

Reply:
It's already on line 134 in the .gitignore file. Do you want me to move it to line 41? I don't think it should make a difference.

Reply:
Ahh OK. If it's there, why has the .idea folder been added in a commit?

Reply:
and now it should be gone!